packages/popover/README.md

```diff
@@ -90,16 +90,15 @@ In pre-v12 versions, a popover can be rendered in 2 ways using the `usePortal` p
+| `dismissMode`                  | `'auto'` \| `'manual'`                    | Options to control how the popover element is dismissed. This will only apply when `usePortal` is undefined and `renderMode` is `'top-layer'` <br> \* `'auto'` will automatically handle dismissal on backdrop click or esc key press, ensuring only one popover is visible at a time <br> \* `'manual'` will require that the consumer handle dismissal manually                     | `'auto'`      |
+| `onToggle`                     | `(e: ToggleEvent) => void;`               | Function that is called when the popover is toggled. This will only apply when `usePortal` is undefined and `renderMode` is `'top-layer'`                                                                                                                                                                                                                                             |               |
+| `portalClassName` (deprecated) | `string`                                  | Passes the given className to the popover's portal container if the default portal container is being used. This will only apply when `usePortal` is `true`                                                                                                                                                                                                                           |               |
+| `portalContainer` (deprecated) | `HTMLElement` \| `null`                   | Sets the container used for the popover's portal. This will only apply when `usePortal` is `true`. <br> NOTE: If using a `scrollContainer` make sure that the `portalContainer` is contained within the `scrollContainer`. E.g, passing the same reference to `scrollContainer` and `portalContainer`.                                                                                |               |
+| `portalRef` (deprecated)       | `string`                                  | Passes a ref to forward to the portal element. This will only apply when `usePortal` is `true`                                                                                                                                                                                                                                                                                        |               |
+| `scrollContainer` (deprecated) | `HTMLElement` \| `null`                   | If the popover portal has a scrollable ancestor other than the window, this prop allows passing a reference to that element to allow the portal to position properly. This will only apply when `usePortal` is `true`                                                                                                                                                                 |               |
```

Comment:
Update the last sentence in each of these to "This will only apply when `renderMode` is `portal`"